typings/objection/index.d.ts

```diff
@@ -1024,4 +1075,95 @@ declare namespace Objection {
 
     QueryBuilderType: QueryBuilder<this, this[]>;
   }
+
+  /**
+   * Overloading is required here until the following issues (at least) are resolved:
+   *
+   * - https://github.com/microsoft/TypeScript/issues/1360
+   * - https://github.com/Microsoft/TypeScript/issues/5453
+   *
+   * @tutorial https://vincit.github.io/objection.js/guide/transactions.html#creating-a-transaction
+   */
+  export interface transaction<T> {
```

Comment:
The `T` type is unused and can be removed.

Reply:
Fixed